ci-operator/step-registry/enable-qe-catalogsource/enable-qe-catalogsource-commands.sh

```diff
@@ -141,8 +163,8 @@ EOF
         node_name=$(oc -n openshift-marketplace get pods -l olm.catalogSource=qe-app-registry -o=jsonpath='{.items[0].spec.nodeName}')
         run_command "oc create ns debug-qe -o yaml | oc label -f - security.openshift.io/scc.podSecurityLabelSync=false pod-security.kubernetes.io/enforce=privileged pod-security.kubernetes.io/audit=privileged pod-security.kubernetes.io/warn=privileged --overwrite"
         run_command "oc -n debug-qe debug node/${node_name} -- chroot /host podman pull --authfile /var/lib/kubelet/config.json ${index_image}"
-        run_command "oc adm node-logs ${node_name} -u kubelet"
-
+        
+        run_command "oc get secret/pull-secret -n openshift-config -o yaml"
```

Comment:
@jianzhangbjz is it safe? once the step is used in test of public repo, the secret will show in public log.

Reply:
Make sense, removed it.